haystack/schema.py

```diff
@@ -2,11 +2,10 @@
 import csv
 import hashlib
 
-import typing
 from typing import Any, Optional, Dict, List, Union
 
 try:
-    from typing import Literal
+    from typing import Literal  # type: ignore
```

Comment:
Why the additional `# type: ignore`? Did mypy fail on this point on the CI?

Reply:
@ZanSara That was a fix for mypy error, on Python 3.7, that doesn't exist. It will fail. The type-ignore is a way around this on CI. The last CI run proves it. 

I'll revert to my original change.